pex/bin/pex.py

```diff
@@ -503,11 +503,18 @@ def make_relative_to_root(path):
   return os.path.normpath(path.format(pex_root=ENV.PEX_ROOT))
 
 
-def main():
+def main(args=None, log_callback=None):
+  if log_callback is not None:
```

Comment:
I'd rather not put mutable objects in as the default options, which is why I went with None.  As far as the sys.argv[1:] being placed in main I'd rather not put that logic somewhere it can't be easily tested.